packages/form-builder/addon/components/cfb-slug-input.js

```diff
@@ -30,6 +31,12 @@ export default class CfbSlugInputComponent extends Component {
 
   @action
   calculatePadding(element) {
+    // Ensures that the element is already visible in combined use
+    // with modal dialogs.
+    scheduleOnce("afterRender", this, this._calculatePadding, element);
```

Comment:
Must be a string, otherwise context binding will be weird.
```suggestion
    scheduleOnce("afterRender", this, "_calculatePadding", element);
```